docs/run/agent/report-hub.md

```diff
@@ -30,7 +30,6 @@ You can customize any of the reports by clicking on its tile.
 Some common options are:
 * `schedule` - a Cron expression describing when to run this report
 * `timeout` - the maximum time this report should run before an error is triggered
-* `save as organization default settings` - if checked, these settings will become your organization's default settings. All new and existing clusters will inherit these settings, unless custom settings have been specified for that cluster.
```

Comment:
??? why did you remove this?

Reply:
How do I make that option appear? I'm logged in as an admin of an organization and it doesn't come up

Reply:
I guess it got removed, I had no idea